src/iframe-attributes.js

```diff
@@ -51,8 +51,7 @@ export function getContextMetadata(
 
   // TODO(alanorozco): Redesign data structure so that fields not exposed by
   // AmpContext are not part of this object.
-  const layoutRect = element.getPageLayoutBox ?
```

Comment:
Removed this method and updated types everywhere to `AmpElement` to avoid this confusion going forward. Thanks for noticing!